lib/core/lib/db/index.js

```diff
@@ -92,24 +57,62 @@ function TransactionClient (client) {
   }
 }
 
-function shutdown (cb) {
-  pool.connect(function (err, client, done) {
-    if (err) return cb(err)
-    client.query('SELECT now()', function (err, result) {
-      if (err) logger.error(err) // log and carry on regardless
-      if (client.release) client.release()
-      pool.end(function (err, done) {
-        if (err) return cb(err)
-        return cb(null, null)
+function buildDb (pool, config) {
+  if (!pool) {
+    /** @see https://github.com/brianc/node-pg-pool#a-note-on-instances */
+    pool = new pg.Pool(config.get('pgdb'))
```

Comment:
For the moment being able to config the db connection is good and the structure can remain like this.

Question here: are there any specific reasons to have the db pool created in core instead of the Udaru service?
Use case: in multi tenant environments it would be needed to have a db pool for each tenant.

Reply:
The idea for me here is to build where it is used. 
If we want to also build it at the service level we can do that and pass it to udaru core